### PR TITLE
update go-sqlite3 version

### PR DESCRIPTION
**Pull request by wbc6080**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
update go-sqlite3 package version
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by ZhengXinwei-F** [comment]
lgtm

**Comment by wbc6080** [comment]
Since the go-sqlite3 official document states that the current new version is after v1.14 and not v2, we need to downgrade the version.

**Comment by wbc6080** [comment]
PLAT @Shelley-BaoYue @fisherxu

File: go.mod [hunk 1]
```diff
@@ -56,7 +56,7 @@ require (
 
 require (
 	github.com/agiledragon/gomonkey v2.0.2+incompatible
-	github.com/beego/beego v1.12.12
```
Comment (fisherxu) [inline go.mod:59]: Will it report error if we don't modify the beego version?

Comment (wbc6080) [reply go.mod:59]: When I downgrade the go-sqlite3 version to 1.14, the go mod automatically downgrades the beego version as well, it looks like go-sqlite3 depends on the 1.10 version of beego

Comment (wbc6080) [reply go.mod:59]: But I found that the latest version of go-sqlite3 is 1.14.18, I will upgrade it to the latest version

Comment (Shelley-BaoYue) [reply go.mod:60]: How about update beego/beego version to v2？

File: go.mod [hunk 2]
```diff
@@ -21,7 +21,7 @@ require (
 	github.com/kubeedge/beehive v0.0.0
 	github.com/kubeedge/viaduct v0.0.0
 	github.com/kubernetes-csi/csi-lib-utils v0.6.1
-	github.com/mattn/go-sqlite3 v2.0.3+incompatible
+	github.com/mattn/go-sqlite3 v1.14.18
```
Comment (Shelley-BaoYue) [inline go.mod:24]: we can update this to newest version 1.14.22

Comment (wbc6080) [reply go.mod]: Now beego/beego version has been upgraded to v2 and go-sqlite3 version has been upgraded to 1.14.22

**Comment by wbc6080** [comment]
Since the v2 version of `beego` is quite different from the v1 version, the unit test cases for sqlite data operations need to be adjusted extensively. In order to avoid blocking the subsequent go and k8s version upgrades, the relevant test cases have been temporarily removed from this PR and use #5677  to track subsequent test case refactoring work @fisherxu @Shelley-BaoYue

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Pull request merged**